web/src/components/home/subcomponents/upcoming_events/upcoming_events.js

```diff
@@ -1,5 +1,92 @@
+import React from "react";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Container from "@mui/material/Container";
+import DyosRenderer from "./event_renderers/dyos_renderer";
+import KeyboardArrowLeft from "@mui/icons-material/KeyboardArrowLeft";
+import KeyboardArrowRight from "@mui/icons-material/KeyboardArrowRight";
+import MobileStepper from "@mui/material/MobileStepper";
+import Typography from "@mui/material/Typography";
+import getNextEvents from "../../../../data/events";
+import FeatureFlags from "../../../../infra/feature_flags";
+import upcomingEventsStyles from "./upcoming_events.styles";
+import messages from "./messages";
+
+function TwoEventRenderer(props) {
```

Comment:
Do you want to cap out at 2? Why not make this display N events and then we can calc how many we want based on the screen size?

Reply:
This is something I wanted to explore via user feedback. I made the interface extensible for getting n events. But for now I wanted to something quick and dirty, and two seemed decent.